common/views/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -496,6 +496,7 @@ const IIIFViewerComponent: FunctionComponent<IIIFViewerProps> = ({
                 />
               </Space>
               <Space
+                data-test-id="rotate-button"
```

Comment:
Is this the right file?

Reply:
I did it in both - I guess this is redundant